library/src/main/java/com/parse/FacebookController.java

```diff
@@ -139,9 +139,9 @@ public void onError(FacebookException e) {
     return authData;
   }
 
-  public void setAuthData(Map<String, String> authData)
+  public void setAuthData(Map<String, String> authData, boolean isFacebookLoginScopeExternallyManaged)
       throws java.text.ParseException {
-    if (authData == null) {
+    if (authData == null && !isFacebookLoginScopeExternallyManaged) {
```

Comment:
Perhaps we should not logout from FB when calling with authData null. What’s the behavior on the iOS repo? We try to have them consistent if possible. Also who calls that with null? When linking fails right?